docs/source/workflow/generate-types.mdx

```diff
@@ -13,47 +13,51 @@ You can use these generated TS types in your resolvers to type-check that your r
 
 ## Setting up your project
 
-We'll use the [GraphQL Code Generator](https://www.the-guild.dev/graphql/codegen) library to generate types based on our GraphQL schema. There are [multiple ways](https://www.the-guild.dev/graphql/codegen/plugins/typescript/typescript-resolvers#usage-example) to provide a schema to GraphQL Code Generator. Below, we'll show one of the common methods, beginning by wrapping our schema in a `gql` template literal tag from the [`graphql-tag`](https://www.npmjs.com/package/graphql-tag) package.
+We'll use the [GraphQL Code Generator](https://www.the-guild.dev/graphql/codegen) library to generate types based on our GraphQL schema. There are [multiple ways](https://www.the-guild.dev/graphql/codegen/plugins/typescript/typescript-resolvers#usage-example) to provide a schema to GraphQL Code Generator. Below, we'll show the most common method, which requires our schema to be in a `.graphql` file.
 
-If you haven't already, begin by installing the `graphql-tag` library:
+If you haven't already, move your server's schema into a `.graphql` file, like so:
 
-```bash
-npm install graphql-tag
-```
+```graphql title="schema.graphql"
+type Query {
+  books: [Book]
+}
 
-The `gql` template literal tag from `graphql-tag` can parse a Schema Definition Language (SDL) string into an abstract syntax tree (AST). Apollo Server accepts a string or an AST as a valid option for defining a schema.  
+type Book {
+  title: String
+  author: String
+}
 
-If we pass GraphQL Code Generator a TypeScript file path, it will automatically generate types for the `gql` tag within that file. So, we can wrap our schema string in a `gql` tag to let GraphQL Code Generator know it should generate types for that tag. 
+type AddBookMutationResponse {
+  code: String!
+  success: Boolean!
+  message: String!
+  book: Book
+}
 
-Import the `gql` tag into the file where you define your schema, then wrap your schema string like so:
+type Mutation {
+  addBook(title: String, author: String): AddBookMutationResponse
+}
+```
 
-```ts title="src/schema.ts"
-import gql from 'graphql-tag';
+If you moved your schema into a `.graphql` file, update your imports to ensure you're still properly passing your schema to your server. In the file where you create your server, you can read in your schema using `readFileSync` from the `fs` package:
 
-export default gql`
-  type Query {
-    books: [Book]
-  }
+```ts title="src/index.ts"
+// ...other imports
+import { readFileSync } from 'fs';
 
-  type Book {
-    title: String
-    author: String
-  }
+// Note: this only works locally because it relies on `npm` routing
```

Comment:
```
// Note: this uses a path relative to the root directory of the project;  this will
// be the current working directory if the server is executed with `npm run`.
```

maybe?

(wanting to point out `npm run` specifically, and not sure "routing" is the right word)

Reply:
I made a slight tweak but kept the overall message! Thank you!

Note: this uses a path relative to the project's root directory, which is the current working directory if the server is executed using `npm run`.

```ts
// Note: this uses a path relative to the project's root directory,
// which is the current working directory if the server is executed using `npm run`.
```